pkg/storage/mysql/mysql.go

```diff
@@ -310,7 +310,7 @@ func (m *MySQL) ReadUsersetTuples(ctx context.Context, store string, filter stor
 		orConditions := sq.Or{}
 		for _, userset := range filter.AllowedUserTypeRestrictions {
 			if _, ok := userset.RelationOrWildcard.(*openfgapb.RelationReference_Relation); ok {
-				orConditions = append(orConditions, sq.Like{"_user": userset.Type + "%#" + userset.GetRelation()})
+				orConditions = append(orConditions, sq.Like{"_user": userset.Type + ":%#" + userset.GetRelation()})
```

Comment:
Can we verify that we have proper indexes to cover this query pattern?

Reply:
<img width="1206" alt="image" src="https://user-images.githubusercontent.com/5374887/222298825-45bdfe9d-f079-4737-9cdd-aaedc4225e76.png">